content/efcore/getting-started/README.md

```diff
@@ -0,0 +1,7 @@
+# Getting Started with Entity Framework Core
+
+## Who Should Read This?
+
+This tutorial is meant for those new to Entity Framework (EF) Core, but if you already have an understanding of EF Core, consider this a refresher course. More advanced functionality is covered in the [docs](https://docs.efproject.net/en/latest/) .
```

Comment:
Could you state that this tutorial covers basic functionality or something like that? It will make the "More advanced functionality..." part make more sense.